test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
@januszm Looking closer at the tests, the from_wkt tests fail for me because, in the comparison at lines 117 and 141, the two factories have different SRIDs. rgeo 0.4 considered the factories equal despite the different SRIDs; rgeo 0.6 considers them different (which makes sense to me). In rgeo 0.6 the second comparison in RGeo::ImplHelper::BasicPointMethods:67 fails. What's going on there in your environment?